core/src/test/java/org/mobilitydata/gtfsvalidator/type/GtfsDateTest.java

```diff
@@ -35,4 +35,43 @@ public void fromString() {
     assertThrows(IllegalArgumentException.class, () -> GtfsDate.fromString("qwerty"));
     assertThrows(IllegalArgumentException.class, () -> GtfsDate.fromString("today"));
   }
+
+  @Test
+  public void getDayMonthYear() {
+    GtfsDate underTest = GtfsDate.fromLocalDate(LocalDate.of(2022, 1, 20));
+    assertThat(underTest.getDay()).isEqualTo(20);
+    assertThat(underTest.getMonth()).isEqualTo(1);
+    assertThat(underTest.getYear()).isEqualTo(2022);
+  }
+
+  @Test
+  public void dateFromEpochShouldReturnGtfsDateWithSameData() {
+    assertThat(GtfsDate.fromEpochDay(18614).getLocalDate()).isEqualTo(LocalDate.ofEpochDay(18614));
+  }
+
+  @Test
+  public void gtfsDateShouldBeComparable() {
+    GtfsDate firstGtfsDate = GtfsDate.fromEpochDay(18614);
+    GtfsDate secondGtfsDate = GtfsDate.fromEpochDay(20614);
+
+    assertThat(firstGtfsDate.compareTo(secondGtfsDate)).isLessThan(0);
+    assertThat(secondGtfsDate.compareTo(firstGtfsDate)).isGreaterThan(1);
+    assertThat(firstGtfsDate.compareTo(firstGtfsDate)).isEqualTo(0);
+  }
+
+  @Test
+  public void firstDateShouldBeIdentifiedAsBeforeSecondDate() {
```

Comment:
Should we also have `secondeDateShouldBeIdentifiedAsAfterFirstDate` ?

Reply:
Per review comment from @aababilov 
> These tests are identical, so you can remove one of them.

https://github.com/MobilityData/gtfs-validator/pull/618#discussion_r562297671